contracts/src/LightClient.sol

```diff
@@ -107,6 +122,13 @@ contract LightClient is Initializable, OwnableUpgradeable, UUPSUpgradeable {
     error InvalidProof();
     /// @notice Wrong stake table used, should match `finalizedState`
     error WrongStakeTableUsed();
+    /// @notice Invalid address
+    error InvalidAddress();
```

Comment:
```suggestion
    error InvalidPermissionedProverAddress();
```
I feel like the name is a bit too non-descriptive now.

Reply:
> I feel like the name is a bit too non-descriptive now.

I made it more general because using the zero address in most cases would not be valid. In the event that the code functionality expands and a check for a non-zero address needs to happen - the same error type can be used. Do you still prefer a very specific error message?

Reply:
I do prefer it to be explicit but it's not exactly strong preference. I just think it makes debugging easier for anyone who ever sees the error. Leave it up to you.